setup.py

```diff
@@ -200,6 +200,7 @@ def run(self):
             "mssql": ["django-pyodbc-azure==1.11.15.0"],
             "mysql": ["mysqlclient==1.3.13"],
             "postgres": ["psycopg2==2.7.5"],
+            "logstash": ["python-logstash==0.46"],
```

Comment:
Shouldn't this be 0.4.6?

```suggestion
            "logstash": ["python-logstash==0.4.6"],
```

Reply:
Yes, it should! Just testing your eagle eyes! :)